powerauth-webflow-authentication/src/main/java/io/getlime/security/powerauth/lib/webflow/authentication/controller/AuthMethodController.java

```diff
@@ -165,7 +154,9 @@ protected UpdateOperationResponse authorize(String operationId, String userId) t
      * @return Response with information about operation update result.
      * @throws NextStepServiceException In case communication fails.
      */
-    protected UpdateOperationResponse authorize(String operationId, String userId, List<KeyValueParameter> params) throws NextStepServiceException {
+    protected UpdateOperationResponse authorize(String operationId, String userId, List<KeyValueParameter> params) throws NextStepServiceException, AuthStepException {
+        // get operation to validate operation state
+        getOperation(operationId);
```

Comment:
Are we just using a side effect here? Can we just validate the operation?

Reply:
Fixed.